app/src/main/java/me/ccrama/redditslide/Activities/MediaView.java

```diff
@@ -1321,133 +1317,6 @@ public void onClick(DialogInterface dialog, int which) {
         });
     }
 
-    private void shareImage(final String finalUrl) {
-        ((Reddit) getApplication()).getImageLoader()
-                .loadImage(finalUrl, new SimpleImageLoadingListener() {
-                    @Override
-                    public void onLoadingComplete(String imageUri, View view, Bitmap loadedImage) {
-                        shareImage(loadedImage, finalUrl);
-                    }
-                });
-    }
-
-    /**
-     * Converts an image to a PNG, stores it to the cache, then shares it. Saves the image to
-     * /cache/shared_image for easy deletion. If the /cache/shared_image folder already exists, we
-     * clear it's contents as to avoid increasing the cache size unnecessarily.
-     *
-     * @param bitmap image to share
-     */
-    private void shareImage(final Bitmap bitmap, String original) {
-        File image; //image to share
-
-        //check to see if the cache/shared_images directory is present
-        final File imagesDir =
-                new File(this.getCacheDir().toString() + File.separator + "shared_image");
-        if (!imagesDir.exists()) {
-            imagesDir.mkdir(); //create the folder if it doesn't exist
-        } else {
-            deleteFilesInDir(imagesDir);
-        }
-
-        try {
-            //creates a file in the cache; filename will be prefixed with "img" and end with ".png"
-            image = File.createTempFile("img", ".png", imagesDir);
-
-            File f = ((Reddit) getApplicationContext()).getImageLoader()
-                    .getDiscCache()
-                    .get(original);
-            if (f != null) {
-                try {
-                    Files.copy(f, image);
-                    shareFile(image);
-                } catch (IOException e) {
-                    doAlternativeImageSave(image, bitmap, original);
-                    shareFile(image);
-                    e.printStackTrace();
-                }
-            } else {
-                doAlternativeImageSave(image, bitmap, original);
-                shareFile(image);
-            }
-        } catch (IOException | NullPointerException e) {
-            e.printStackTrace();
-            Toast.makeText(this, getString(R.string.err_share_image), Toast.LENGTH_LONG).show();
-        }
-    }
-
-    public void shareFile(File image) {
```

Comment:
Dropping this method entirely did not seem to change if sharing succeeded from MediaView. What additional work was this method doing. If something meaningful, should it be re-added so AlbumPager can leverage it too?